cmdstanpy/model.py

```diff
@@ -524,7 +536,7 @@ def sample(
                                 tqdm_pbar = tqdm.tqdm_notebook
                             except ImportError:
                                 msg = (
-                                    "'tqdm_notebook' could not be imported. "
+                                    'Cannot import package tqdm_notebook.\n'
```

Comment:
It tries to import `tqdm.tqdm_notebook` module, but I think this message is ok.

Reply:
will make it explicit 'tqdm.tqdm_notebook'